actors/builtin/shared.go

```diff
@@ -87,3 +87,8 @@ func ResolveToIDAddr(rt runtime.Runtime, address addr.Address) (addr.Address, er
 
 	return idAddr, nil
 }
+
+type ApplyRewardParams struct {
+	Reward  *abi.TokenAmount
+	Penalty *abi.TokenAmount
```

Comment:
Neither of these should be pointers here. 

(When the single TokenAmount was the parameter, it needed to be a pointer to satisfy the CBORMarshaller interface)